### ISSUE TITLE
Detail & Search Panels not cleared 

### ISSUE THREAD

**Issue by Jiros** [covidgraph/visual-graph-explorer#32]
The detail panel and any existing search values are not cleared when I use the clear graph 'bin' icon which is what I was expecting to happen. i.e. to start with a clean slate.

**Comment by yGuy** [comment]
Good point about the detail panel. However I would argue that keeping the previous search results in the search values can actually be very helpful, because you can start with similar search (same genes, but different papers, authors, etc.). At least I used that extensively during testing, but that was just -well- testing and not really using it.

**Comment by Jiros** [comment]
Also a good point - if I remember from yesterday's discussion one of the ideas was to be able to save and load queries? 

Or possibly a second 'clear Searches' icon somewhere?

**Comment by yGuy** [comment]
The integrated "omni-search" doesn't have the stale search data problem anymore, I'd say. The detail panel bug is still there, though.
